lib/wpc_proto/wpc_proto.c

```diff
@@ -33,6 +33,80 @@
 // Max possible size of encoded message
 #define MAX_PROTOBUF_SIZE WP_CONFIG_MESSAGE_PB_H_MAX_SIZE
 
+static onEventStatus_cb_f m_onProtoEventStatus_cb = NULL;
+
+
+static void onStackStatusReceived(uint8_t status)
```

Comment:
Why not moving directly this code to proto_config module directly?
This is where it belongs, no?

But we can do it later on